--- skype-1.1.0.3/skype.desktop~	2005-05-18 08:19:41.000000000 +0200
+++ skype-1.1.0.3/skype.desktop	2005-05-18 08:19:41.000000000 +0200
@@ -1,9 +1,10 @@
 [Desktop Entry]
 Name=Skype
-Comment=Skype Internet Telephony
+Comment=p2p VoIP application
+Comment[pl]=Aplikacja VoIP p2p
 Exec=skype
 Icon=skype.png
-Terminal=0
+Terminal=false
 Type=Application
 Encoding=UTF-8
-Categories=Network;Application;
+Categories=Qt;Network;InstantMessaging;
